implementations/rust/Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
     "kex",
     "message",
     "vault",
-    "router",
-    "transport",
+    # "router",
```

Comment:
what's the reason these are commented out?

Reply:
Not checked in, in this PR.

Reply:
...meaning, it wasn't meant to be part of this PR